### DISCUSSION TITLE
Network 패킷 지연 현상

### DISCUSSION THREAD

**Discussion by strangestraw** [naverz/zepeto-studio-kor#392] [Issue Report (Legacy)]
안녕하세요.
멀티플레이를 통한 유저간 상태 동기화 작업을 진행 중인데,
보내는 패킷에 대한 처리 지연(패킷이 늦게 오거나), 패킷 loss가 발생하는 상태입니다.
제페토 멀티플레이 서버에 문제가 있는 것 같은데,
관련해서 문제 파악여부나 개선 스케줄 등이 있으신 지 궁금합니다.
확인 부탁드립니다.
감사합니다.

**Comment by hyeonjin-jo** [comment]
안녕하세요.

사용하신 코드 공유 및 상황에 대한 조금 더 상세한 설명을 해주시면 내용 파악에 도움이 될 것 같습니다
